tests/phpunit/test-register-meta.php

```
@@ -0,0 +1,55 @@
+	 */
+	public function test_pad_meta_show_in_rest() {
+		$meta_keys = get_registered_meta_keys( 'post', 'post' );
+		$this->assertArrayHasKey( 'pad_hide_post_author', $meta_keys );
+		$this->assertTrue( $meta_keys['pad_hide_post_author']['show_in_rest'] );
+	}
+
+	/**
+	 * Test that meta value can be saved and retrieved.
+	 */
+	public function test_meta_save_and_retrieve() {
+		$post_id = $this->factory->post->create();
```

PHPStan (2.1.46)

[error] Access to an undefined property RegisterMetaTest::$factory.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)